cloudinit/sources/DataSourceEc2.py

```diff
@@ -508,21 +499,6 @@ def network_config(self):
             return None
 
         result = None
-        no_network_metadata_on_aws = bool(
```

Comment:
This looks like a fine change, but I think it should be its own commit or PR.

Reply:
See https://github.com/canonical/cloud-init/pull/5714